templates/ptpd.conf.erb

```diff
@@ -0,0 +1 @@
+
```

Comment:
There is nothing in the template. This should be a different version of the config file.